scripts/triggered_mails.py

```diff
@@ -0,0 +1,43 @@
+import sys
+import logging
+from datetime import datetime, timedelta
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+from website.app import init_app
+from website import mails, settings
+from framework.auth import User
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+def main(dry_run=True):
+    for user in find_inactive_users_with_no_inactivity_email_sent_or_queued():
+        if dry_run:
+            logger.warn('Dry run mode')
+        logger.warn('Email of type no_login queued to {0}'.format(user.username))
+        if not dry_run:
+            with TokuTransaction():
+                mails.queue_mail(
+                    to_addr=user.username,
+                    mail=mails.NO_LOGIN,
+                    send_at=datetime.utcnow(),
```

Comment:
so that it can go through the week_check and potentially be sent next week, and so the presend can be called on it.